packages/atomic/cypress/integration/pager.cypress.ts

```diff
@@ -122,6 +122,15 @@ describe('Pager Test Suites', () => {
       cy.get('@pagerLi').get(PagerSelectors.buttonPrevious).should('not.exist');
     });
 
+    it('should load without navigation buttons when user goes to page 2', () => {
```

Comment:
Either the test description is unclear or I am confused about the test.

Why does the navigation disappear on page 2? Shouldn't it be absent on all pages when `enable-navigation-buttons=false` is configured? What is special about page 2 in this case?

Reply:
Normally when you load the page, the 'previous' button won't be visible, so we go to page 2 is to make sure that even if we activate/use the 'pager', the "previous button" won't be visible at all